backend/doc/cli.md

```diff
@@ -32,16 +32,110 @@ This will drop the entire database, be careful!
 
 ### add a user
 
-    tracimcli user create -l "john@john@john.john" -p "superpassword"
+```
+$ tracimcli user create -h
+usage: tracimcli user create [-h] [-c CONFIG_FILE] [-d] [-e EMAIL]usage: tracimcli user create [-h] [-c CONFIG_FILE] [-d] [-e EMAIL]
+                             [-u USERNAME] [--public-name PUBLIC_NAME]
+                             [--allowed_space ALLOWED_SPACE] [--lang LANG]
+                             [-p PASSWORD] [--profile PROFILE]
+                             [--timezone TIMEZONE] [--send-email]
+
+Create user
+
+optional arguments:
+  -h, --help            show this help message and exit
+  -c CONFIG_FILE, --config CONFIG_FILE
+                        application config file to read (default:
+                        development.ini)
+  -d, --debug_mode      enable Tracim log for debug
+  -e EMAIL, --email EMAIL
+                        set user email
+  -u USERNAME, --username USERNAME
+                        set username of the user
+  --public-name PUBLIC_NAME
+                        set user public name
+  --allowed_space ALLOWED_SPACE
+                        set user allowed space in bytes
+  --lang LANG           set user lang (ISO 639 format)
+  -p PASSWORD, --password PASSWORD
+                        set user password
```

Comment:
```suggestion
                        set the password of the user
```

Reply:
why not "set the user's password" , to have similar sentence everywhere ?